jetpack/jetpack-start-endpoints/upgrade-paths.md

```diff
@@ -0,0 +1,60 @@
+# Upgrade Paths
+
+Not all Jetpack product offerings should be provisioned concurrently. Some products contain the functionality of others so it doesn't make sense to provision them concurrently. The logic to determine which products can be provisioned for a particular site is quite complex and should not be implemented by partners. This is exactly what the upgrade paths endpoint is about. In short, given a blog ID or a URL, it will return what products can be provisioned.
+
+The only time when we allow products with overlapping functionality to be provisioned is when the product to be provisioned is a superset of a product already in place. For example, Jetpack Professional includes Realtime Backups as part of its functionality. Consider these two scenarios:
+
+1. A site already has Realtime Backups provisioned. A hosting partner wants to provision Jetpack Professional. Even though Jetpack Professional already includes Realtime Backups, this action *will* be allowed as the site would be gaining extra functionality.
+2. A site already has Jetpack Professional provisioned. A hosting partner wants to provision Realtime Backups. This *will not* be allowed because the site would not gain anything from provisioning Realtime Backups as that same functionality is already included in the existing Jetpack Professional plan. 
+
+### Endpoint information (/provision)
+
+- __Method__: GET
+- __URL__:    `https://public-api.wordpress.com/rest/v1.3/jpphp/{$blog_id}/upgrade-paths`
```

Comment:
We allow passing either the blog ID or site slug now, right?

Reply:
Yes sir. Changed it so that is now says {$siteOrBlogId} like the status endpoint (I copied and pasted the same explanation that was in there).